build-tools/tasks/docs.js

```diff
@@ -13,41 +12,24 @@ module.exports = function docs() {
   return Promise.resolve();
 };
 
-const publicDirs = listPublicItems('src');
-
-function validatePublicFiles(definitionFiles) {
```

Comment:
Not needed anymore, because documenter now validates it internally: https://github.com/cloudscape-design/documenter/blob/main/src/components/extractor.ts#L139